tests/test_sourceSelector.py

```diff
@@ -187,6 +187,7 @@ def setUp(self):
         self.config.magLimit.fluxField = "flux"
         self.config.doMagLimit = True
         self.config.doFlags = True
+        self.config.doUnresolved = False
```

Comment:
Isn't this the default?

Reply:
It is, but I'm following the redundancy in the ScienceSourceSelectorTask test.